### DISCUSSION TITLE
v1.3.0

### DISCUSSION THREAD

**Discussion by akshayp7** [akshayp7/playwright-typescript-playwright-test#52] [Announcements]
Dedicated to J. Robert Oppenheimer : American Theoretical Nuclear physicist

Changelog:
1. Updated to Playwright 1.35.1
2. Changed the way of reading environment variables instead of using cross-env library, now we are using "process.env" variable. Please refer Readme section for more info 

<hr /><em>This discussion was created from the release <a href='https://github.com/akshayp7/playwright-typescript-playwright-test/releases/tag/v1.3.0'>v1.3.0</a>.</em>

**Comment by Zarielos** [comment]
Hey, I am currently trying to run the new version on local env and I'm having a problem with process.env, no matter how I try to use it from CMD it says "Please provide a correct env value like....". Is something missing from README? I see that gitignore has a .env file excluded, could you please take a look into this when you have some time? I've checked 1.2.0 version and it works just fine.

**Reply by HamzaMiz** [reply]
Hello , Same as you @Zarielos  i was gonna open a question and i found your comment , i tried this. npx playwright test Login.test.ts --project=Chrome --ui --ENV="qa" i got Please provide a correct environment value like "npx cross-env ENV=qa|dev|qaApi|devApi" , and please whats the use of ENV ? , thanks for your effort

**Reply by akshayp7** [reply]
Thanks for your patience ENV is used for Environment variable . For e.g if you have to run the tests on QA environment you can set the environment url in "testConfig.ts" file similarly for dev also you can set variable as dev and give url in testconfig file. While running if you can choose between dev and qa url in command line using ENV

**Comment by akshayp7** [comment]
Previously I was using external library for env variables .  Now I have removed the library and using proccess.env
